src/test/java/org/takes/rs/RsPrintTest.java

```diff
@@ -70,6 +75,29 @@ public void flushBodyEvenWhenExceptionHappens() throws IOException {
         );
     }
 
+    /**
+     * RsPrint can close body contents even when exception happens.
+     * @throws IOException If some problem inside
+     */
+    @Test
+    public void closeBodyEvenWhenExceptionHappens() throws IOException {
+        final IOException exception = new IOException("Smth went wrong");
+        final FailOutputStream output = new FailOutputStream(exception);
+        final FakeInput input = new FakeInput(new InputStreamOf("abc"));
+        try {
+            new RsPrint(new RsText(input))
+                .printBody(output);
+        } catch (final IOException ex) {
+            if (!ex.equals(exception)) {
+                throw ex;
+            }
+        }
+        MatcherAssert.assertThat(
```

Comment:
@olenagerasimova it's a good practice to add description text to `assertThat` if we have more than one assertions in test.